run.sh

```diff
@@ -1,69 +1,95 @@
 #!/usr/bin/env bash
 BASE_DIR=$(cd $(dirname $0); pwd -P)
-CASE_FILE=$2
-#export RUST_LOG=debug
+CASE_FILE="cases.json"
+MODE=""
 
-run_boring_server() {
-  cargo run -- \
-  --client $BASE_DIR/../dist/Debug/bin/nss_bogo_shim \
-  --server $BASE_DIR/../boringssl/build/ssl/test/bssl_shim \
-  --rootdir $BASE_DIR/../boringssl/ssl/test/runner/ \
-  --test-cases $BASE_DIR/$CASE_FILE \
-  --client-writes-first
-}
+while [ $# -gt 0 ]; do
+    case $1 in
+        -v) export RUST_LOG=debug ;;
+        -m) MODE="$2"; shift ;;
+        -c) CASE_FILE="$2"; shift;;
+        *) echo "Error: Unknown argument."; exit 2 ;;
+    esac
+    shift
+done
 
-run_boring_client() {
-  cargo run -- \
-  --client $BASE_DIR/../boringssl/build/ssl/test/bssl_shim \
-  --server $BASE_DIR/../dist/Debug/bin/nss_bogo_shim \
-  --rootdir $BASE_DIR/../boringssl/ssl/test/runner/ \
-  --test-cases $BASE_DIR/$CASE_FILE
-}
+while [ $# -gt 0 ]; do
```

Comment:
Why doing this twice? :upside_down_face:

Reply:
Just to be sure. ;)
I removed the duplication.